doc/external_resources.md

```diff
@@ -12,6 +12,16 @@ to submit a pull request with your recommended addition to the
 
 :::::{grid} 1 2 2 3
 
+::::{grid-item-card} 2024 PyGMT Webinar using Google Colab (in Portuguese)
+:link: [[https://www.generic-mapping-tools.org/egu22pygmt](https://github.com/andrebelem/Oficina_PyGMT)/](https://github.com/andrebelem/Oficina_PyGMT)
```

Comment:
Humm...how can I change this ? The construction is wrong. It should be:
link: [https://github.com/andrebelem/Oficina_PyGMT](https://github.com/andrebelem/Oficina_PyGMT)

Reply:
This looks like my suggestion from the code review. Are you able / allowed to click the "Commit suggestion" button, which is just below my inital review comment? Then you can apply my suggestion directly:

![github_review_commit_suggestion](https://github.com/user-attachments/assets/6a39b8fd-f56b-4700-809b-dfc924c0e1b4)

Of course you can also make a new commit with the correct link and push to this branch.

Reply:
Yeap. I did it.

Reply:
Great! Should be good now https://pygmt-dev--3360.org.readthedocs.build/en/3360/external_resources.html.

Reply:
Please update the title to make it more descriptive.

Reply:
I am sorry, my comment was probably confusing 🙁. It's only about the title of the PR. You can change it via the "edit" bottom on the right:
![title_pr](https://github.com/user-attachments/assets/dad4db53-9a69-487c-8cf2-81ee90835c2e)

Maybe something like "External Resources: Add tutorial in Portuguese "

Reply:
Perfect. I changed the title. Thanks.